salesagility/SuiteCRM
# Studio: Adding templateMeta to Custom List View

task: scrm-591
issue: #3628

## Description



When a user edits the list view layouts, the custom buttons disapear, due to studio failing to copy the templateMeta key into the custom version of the listviewdiefs.

## Motivation and Context

This change adds the ability in studio to copy the templateMeta key into the listviewdefs.

## How To Test This

* Repair and Rebuild
* Go into Studio -> emails -> layouts -> list view
*  Save and deploy
* Check the emails list view

## Types of changes


### Final checklist
